src/sequencing.js

```diff
@@ -0,0 +1,29 @@
+import { always, converge } from 'ramda';
+
+/* eslint-disable max-len */
+/**
+ * Accepts a list of side-effecting functions and returns a new function. When
+ * invoked, this new function is applied to some arguments, each side-effecting
+ * function is applied to those same arguments in the order in which they were
+ * declared. The return of each side-effecting function is discarded and the
+ * new function will always return `undefined`.
+ *
+ * @func sequencing
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.3.0|v2.3.0}
+ * @category Function
+ * @sig [((a, b, ...) -> x | undefined), ((a, b, ...) -> y | undefined), ...] -> (a -> b -> ... -> undefined)
+ * @param {Array} functions A list of side-effecting functions.
```

Comment:
```js
@param {Array} fns A list of side-effecting functions
```
Rename `functions` to `fns` to we are consistent with typescript and drop the dot at the end of the line

Reply:
Followed Ramda, but will rename.